gh-88013: ntpath.realpath may raise TypeError in some cases (#102813)
python/cpython: Misc/NEWS.d/next/Windows/2023-03-18-21-38-00.gh-issue-88013.Z3loxC.rst

> Fixed a bug which will cause ``TypeError`` when calling
> ``ntpath.realpath()`` with bytes parameter in some cases.

Reviewer: ```suggestion
Fixed a bug where :exc:`TypeError` was raised when calling
:func:`ntpath.realpath` with a bytes parameter in some cases.
```
